Sources/CardPayments/CardClient.swift

```diff
@@ -44,6 +44,8 @@ public class CardClient: NSObject {
     /// - Parameters:
     ///   - vaultRequest: The request containing setupTokenID and card
     public func vault(_ vaultRequest: CardVaultRequest) {
+        analyticsService = AnalyticsService(coreConfig: config, setupToken: vaultRequest.setupTokenID)
+        analyticsService?.sendEvent("card-payments:vault-wo-purchase:started")
```

Comment:
Should we extract analytics out into a constants file on PPCP: [example from the BT SDK](https://github.com/braintree/braintree_ios/blob/main/Sources/BraintreeCard/BTCardAnalytics.swift)?

Reply:
This is a good call. Making DTPPCPSDK-1984 to address this project wide.